q2_intervention/_utilities.py

```diff
@@ -27,28 +27,40 @@
 TEMPLATES = pkg_resources.resource_filename('q2_intervention', 'assets')
 
 
+def _check_inputs(state_1, state_2):
```

Comment:
Could you rename this, maybe `_validate_state_values`, to be a little more self-documenting?